pkg/reconciler/revision/table_test.go

```diff
@@ -173,7 +174,8 @@ func TestReconcile(t *testing.T) {
 		// are necessary.
 		Objects: []runtime.Object{
 			rev("foo", "stable-reconcile", WithLogURL, AllUnknownConditions),
-			pa("foo", "stable-reconcile", WithReachabilityUnknown),
+			pa("foo", "stable-reconcile", WithReachability(asv1a1.ReachabilityUnknown)),
```

Comment:
This doesn't feel like an improvement?

Reply:
You had three methods, not there's one. So, it feels like an improvement.
What is really concerning, is that there are zero unit tests that covered the not "Unknown" state...

Reply:
> You had three methods, not there's one. So, it feels like an improvement.

This is a test, where we should prioritize clarity and readability over saving lines of code.

Reply:
This is extremely subjunctive. I think it's extremely clear what is happening here and should I need to change/replicate the test, it will be easier.